File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWith2DBarcodes/Basic2DBarcodeFeatures/CreatingPdf417Barcode.php
<?php
namespace Aspose\Barcode\WorkingWith2DBarcodes\Basic2DBarcodeFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;

class CreatingPdf417Barcode{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $builder = new BarCodeBuilder();

        $symbology=new Symbology();
        $builder->setSymbologyType($symbology->Pdf417);

        $builder->setCodeText("12345");

        # Save the image to your system and set its image format to Jpeg
        $builder->save($dataDir . "CreatingPdf417Barcode.jpg");

        # Display Status
        print "Created Pdf417 Barcode Successfully.".PHP_EOL;

    }


}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWith2DBarcodes/Basic2DBarcodeFeatures/CreatingPdf417Barcode.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology

class CreatingPdf417Barcode:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWith2DBarcodes/Basic2DBarcodeFeatures/CreatingPdf417Barcode/'

        # Instantiate barcode object
        builder =  BarCodeBuilder()

        symbology= Symbology
        builder.setSymbologyType(symbology.Pdf417)

        builder.setCodeText("12345")

        # Save the image to your system and set its image format to Jpeg
        builder.save(dataDir + "CreatingPdf417Barcode.jpg")

        # Display Status
        print "Created Pdf417 Barcode Successfully."
        
if __name__ == '__main__':        
    CreatingPdf417Barcode()